scripts/robust.yaml

```diff
@@ -0,0 +1,62 @@
+id: str(min=7, max=7)
+title: str(min=1, max=160)
+description: str(min=1)
+classification: str(min=1)
+keywords: list(str(min=1, max=80))
+system: str(min=1, max=40)
+severity: enum("error", "warning", "convention-violation", "bad-smell",
+            "minor-issue", "not-a-bug")
+links: list(str())
```

Comment:
Fix this

Reply:
added a preliminary check for URLs in https://github.com/robust-rosin/robust/pull/359/commits/9e9be82a385fae1ee0e286c7048645fc87632cc8, but better solutions welcome